--- init.rc	2012-11-30 12:55:59.208578163 +0100
+++ init1.rc	2012-11-30 13:00:01.493652787 +0100
@@ -37,6 +37,10 @@
     export LOOP_MOUNTPOINT /mnt/obb
     export BOOTCLASSPATH /system/framework/core.jar:/system/framework/core-junit.jar:/system/framework/bouncycastle.jar:/system/framework/ext.jar:/system/framework/framework.jar:/system/framework/framework2.jar:/system/framework/android.policy.jar:/system/framework/services.jar:/system/framework/apache-xml.jar:/system/framework/sec_edm.jar:/system/framework/seccamera.jar
 #WTL_EDM
+
+# Preliminary fix for exynos-mem exploit (for more details, see http://forum.xda-developers.com/showthread.php?p=35560095)
+    chmod 440 /dev/exynos-mem
+
 # Backward compatibility
     symlink /system/etc /etc
     symlink /sys/kernel/debug /d
@@ -763,13 +763,6 @@
     disabled
     oneshot
 
-service samsungani /system/bin/samsungani
-    class main
-    user graphics
-    group graphics system
-    disabled
-    oneshot
-
 service dbus /system/bin/dbus-daemon --system --nofork
     class main
     socket dbus stream 660 bluetooth bluetooth
@@ -883,3 +876,10 @@
 on property:init.svc.media=restarting
     exec icd_check
     start icd
+
+# Call Boeffla-Kernel init script
+service boeffla-init /sbin/boeffla-init.sh
+    class core
+    user root
+    oneshot
+

